### PR TITLE
docs: Add section how to install Agent on Linux

### LINKED ISSUE #844: Host deb/rpms in Grafana apt/yum repositories
Grafana has an [APT](https://grafana.com/docs/grafana/latest/installation/debian/#install-from-apt-repository) and [YUM](https://grafana.com/docs/grafana/latest/installation/rpm/#install-from-yum-repository) repository hosting builds of Grafana. It would be great to include the Grafana Agent builds there too. 

I've found an [update_repo](https://github.com/grafana/grafana/tree/main/scripts/build/update_repo) directory in Grafana, but it's not clear if that's still used for updating the repositories. 

This change likely need to be done by a Grafana Labs employee due to the usage of secrets for uploading the artifacts and signing the package.

### PR DESCRIPTION
**Pull request by marctc**
#### PR Description

This PR adds a section in the docs explaining how to install the Agent
using Linux package managers either for deb or rpm packages.

#### Which issue(s) this PR fixes

Closes #844

#### Notes to the Reviewer

I couldn't test for RPM-based Linux but I tested for deb and it
worked.

#### PR Checklist

- [ ] CHANGELOG updated
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Review by tpaschalis — COMMENTED** [review]
LGTM overall.

One point we could consider expanding upon; when we install the Agent via the repos I think we're setting it up as a systemd service, right?

If so, could we add a note on where they can find their default configuration file (I _think_ it's `/etc/grafana-agent.yaml`), how to reload that config, how to see the Agent's status, logs etc?

When we install the Agent via the [helper script](https://github.com/grafana/agent/blob/main/production/grafanacloud-install.sh) in Grafana Cloud, we at least log a few useful lines to their stdout.

**Review by tpaschalis — APPROVED** [review]

**Comment by marctc** [comment]
> LGTM overall.
> 
> One point we could consider expanding upon; when we install the Agent via the repos I think we're setting it up as a systemd service, right?
> 
> If so, could we add a note on where they can find their default configuration file (I _think_ it's `/etc/grafana-agent.yaml`), how to reload that config, how to see the Agent's status, logs etc?

I din't know about that, good point! I will add it

File: docs/sources/set-up/install-agent-linux.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+### Operational guide
+
+The Grafana Agent will be configured a systemd service after using the installation methods
+explained in the previous sections.
+
+#### Start the Agent
+
+To run the service you just need to type:
+```shell
+sudo service grafana-agent start
```
Comment (tpaschalis) [inline docs/sources/set-up/install-agent-linux.md:66]: I think systemd has these arguments the other way, right? Like `sudo systemctl start grafana-agent`.

Did this work on your machine?

File: docs/sources/set-up/install-agent-linux.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```shell
+sudo service grafana-agent status
+```
+
+#### Editing the Agent's config file
+
+By default, the config file is located in `/etc/grafana-agent.yaml`. After editing the file
+with the desired config, you need to restart the agent running:
+```shell
+sudo service grafan-agent restart
```
Comment (tpaschalis) [inline docs/sources/set-up/install-agent-linux.md:79]: s/grafan/grafana

File: docs/sources/set-up/install-agent-linux.md (hunk 1 above)
Comment (tpaschalis) [reply docs/sources/set-up/install-agent-linux.md:66]: In any case, I'm not siding with any specific way of configuring services, just pointing out that in grafanacloud-install.sh we prompt people to use systemctl.

Comment (marctc) [reply docs/sources/set-up/install-agent-linux.md]: Either works. I will use `systemctl` for coherence with other docs.

**Review by tpaschalis — APPROVED** [review]

**Pull request merged**